root/components/LinkSearchableProperty.js

```diff
@@ -32,7 +32,8 @@ const LinkSearchableProperty = ({
   const url = new URL($c.req.uri);
   url.pathname = '/search';
   url.search =
-    'query=' + encodeURIComponent(searchField + ':' + searchValue) +
+    'query=' +
+    encodeURIComponent(searchField + ':"' + searchValue + '"') +
```

Comment:
I think you should wrap `searchValue` in `escapeLuceneValue`: root/static/scripts/common/utility/escapeLuceneValue.js